frontend/src/pages/about/PageOutreach.vue

```diff
@@ -6,60 +6,108 @@
   <AppSection>
     <AppHeading>Outreach</AppHeading>
     <p>
-      The Monarch Initiative is a community effort, and we welcome your participation!
+      The Monarch Initiative is a community effort, and we welcome your
```

Comment:
The section component isn't intended to be nested. You should close the first section here then start a new one.